src/cegis/jsa/genetic/jsa_source_provider.cpp

```diff
@@ -26,7 +26,7 @@ jsa_source_providert::jsa_source_providert(jsa_symex_learnt &lcfg) :
 {
 }
 
-#define START_METHOD_PREFIX "void _start"
+#define START_METHOD_PREFIX "void __CPROVER_start"
```

Comment:
Does it make sense to put this into util/prefix.h to really have it in a single place?